extra/exe-builder/Program.cs

```diff
@@ -173,7 +190,11 @@ public class UptimeKumaApplicationContext : ApplicationContext
         }
 
         void CheckForUpdate(object sender, EventArgs e) {
-            var needUpdate = false;
+            if (intranetOnly)
+            {
+                MessageBox.Show("You have parsed in --intranet so we will not try to access the internet.");
```

Comment:
`The --intranet argument was provided, so we will not try to access the internet.` should already be in the log => I don't see why logging this again has additional value

```suggestion
```